src/ORM/Behavior/TreeBehavior.php

```diff
@@ -76,6 +77,23 @@ class TreeBehavior extends Behavior
     ];
 
     /**
+     * {@inheritDoc}
+     */
+    public function __construct(Table $table, array $config = [])
+    {
+        parent::__construct($table, $config);
+        $connection = $table->connection();
+        if ($connection !== null) {
+            $driver = $connection->driver();
+            $this->config('leftField', $driver->quoteIdentifier($this->config('left')));
+            $this->config('rightField', $driver->quoteIdentifier($this->config('right')));
+        } else {
+            $this->config('leftField', $this->config('left'));
+            $this->config('rightField', $this->config('right'));
```

Comment:
Won't configuration set at runtime run into the same quoting issue? Should we be quoting when expressions are built instead?